lib/network/modules/InteractionHandler.js

```diff
@@ -165,8 +165,13 @@ class InteractionHandler {
 
   onContext(event) {
     let pointer = this.getPointer({x:event.clientX, y:event.clientY});
-    this.checkSelectionChanges(pointer, event);
-    this.selectionHandler._generateClickEvent('oncontext', event, pointer);
+    const hoveredElements = {
+      nodes: (this.selectionHandler.getNodeAt(pointer) || []),
+      edges: (this.selectionHandler.getEdgeAt(pointer) || [])
+    }
+    hoveredElements.nodes = [].concat(hoveredElements.nodes)
+    hoveredElements.edges = [].concat(hoveredElements.edges)
+    this.selectionHandler._generateClickEvent('oncontext', Object.assign({}, event, hoveredElements), pointer);
```

Comment:
One I should have spotted earlier: `nodes` and `edges` are defined here as part of the event. In the output, they are not. `_generateClickEvent()` create the `properties` object, adds `nodes` and `edges` to it, and the original event (which you are changing) is made a *member* of this `properties` object.